docs/esp8266/general.rst

```diff
@@ -183,13 +184,16 @@ limitation with usage of TLS on the low-memory devices:
    accessing various REST APIs, which usually require much smaller messages.
    The buffers size is on the order of 5KB, and is adjusted from time to
    time, taking as a reference being able to access https://google.com .
-   The smaller buffer hower means that some sites can't be accessed using
-   it, and it's not possible to stream large amounts of data.
+   The smaller buffer however means that some sites can't be accessed using
+   it, and it's not possible to stream large amounts of data. axTLS does
+   have support for TLS's Max Fragment Size extension, but no HTTPS website
+   does, so use of the extension is really only effective for local
+   communication with other devices.
 
 There are also some not implemented features specifically in MicroPython's
 ``ussl`` module based on axTLS:
 
-6. Certificates are not validated (this may make connections susceptible
+6. Certificates are not validated (this makes connections susceptible
```

Comment:
Again on purpose. There's modality in "susceptible", but who's linguist in the real world? I don't want to needlessly alert users that giving associations like "there's always mad in the middle in there". Otherwise, TLS connections are always susceptible to man in the middle attacks due to CAs, super- and quantum computers.